client/templates/includes/gillespyResultsEnsemble.pug

```diff
@@ -39,25 +39,25 @@ div#workflow-results-ensemble.card.card-body
       div
 
         h5.inline Plot Trajectories
-        button.btn.btn-outline-collapse(data-toggle="collapse", data-target="#collapse-trajectories", data-hook="collapse-trajectories") +
+        button.btn.btn-outline-collapse(data-toggle="collapse", data-target="#collapse-trajectories", id="collapse-trajectories", data-hook="collapse-trajectories") +
 
       div.collapse(id="collapse-trajectories")
 
-        div(data-hook="trajectories")
+        div(id="trajectories", data-hook="trajectories")
         button.btn.btn-primary.box-shadow(id="trajectories" data-hook="plot") Plot
-        button.btn.btn-primary.box-shadow(data-hook="plot-multiple", disabled) Multiple Plots
+        button.btn.btn-primary.box-shadow(id="plot-multiple", data-hook="plot-multiple", disabled) Multiple Plots
         button.btn.btn-primary.box-shadow(id="trajectories" data-hook="download-png-custom" disabled) Download PNG
         button.btn.btn-primary.box-shadow(id="trajectories" data-hook="download-json" disabled) Download JSON
 
     div.card.card-body
       div
 
         h5.inline Plot Standard Deviation
-        button.btn.btn-outline-collapse(data-toggle="collapse", data-target="#collapse-stddev", data-hook="collapse-stddev") +
+        button.btn.btn-outline-collapse(data-toggle="collapse", data-target="#collapse-stddev", id="collapse-stddev", data-hook="collapse-stddev") +
```

Comment:
The element on line 58 already has this id value I would suggest adding '-btn' to the end of the existing id value.

No element use my suggested id value